src/Files.App/UserControls/StatusBarControl.xaml

```diff
@@ -137,9 +138,37 @@
 				Visibility="{x:Bind SelectedItemsPropertiesViewModel.ItemSizeVisibility, Mode=OneWay}" />
 		</StackPanel>
 
+		<!--  Open in VS Code Button  -->
+		<Button
+			x:Name="OpenInVSCodeButton"
+			Grid.Column="1"
+			Height="24"
+			Padding="8,0,8,0"
+			VerticalAlignment="Center"
+			x:Load="{x:Bind converters:MultiBooleanConverter.AndNotConvert(Commands.OpenInVSCode.IsExecutable, Commands.OpenInVS.IsExecutable), Mode=OneWay}"
+			Background="Transparent"
+			BorderBrush="Transparent"
+			Command="{x:Bind Commands.OpenInVSCode}"
+			Content="{x:Bind Commands.OpenInVSCode.Label}"
+			ToolTipService.ToolTip="{x:Bind Commands.OpenInVSCode.LabelWithHotKey, Mode=OneWay}" />
+
+		<!--  Open in VS Button  -->
+		<Button
+			x:Name="OpenInVSButton"
+			Grid.Column="1"
+			Height="24"
+			Padding="8,0,8,0"
+			VerticalAlignment="Center"
+			x:Load="{x:Bind Commands.OpenInVS.IsExecutable, Mode=OneWay}"
+			Background="Transparent"
+			BorderBrush="Transparent"
+			Command="{x:Bind Commands.OpenInVS}"
+			Content="{x:Bind Commands.OpenInVS.Label}"
+			ToolTipService.ToolTip="{x:Bind Commands.OpenInVS.LabelWithHotKey, Mode=OneWay}" />
```

Comment:
Should these be in the StackPanel

Reply:
It depends: at the moment only one will be visible, never both at the same time. In this case there's no need for the `StackPanel`. If you think that we should show them both (if all the requirements are satisfied), then I'll add it

Reply:
The StackPanel will handle the spacing, I think it's worth doing this.

Reply:
> If you think that we should show them both

I don't know of a common use case for showing both at the same time.

Reply:
The `Open in VS` button should be visible even if it's not a git repo (but it has a `.sln` file), am I right?

Reply:
I think only in a git repo because we anyways have buttons in the status bar.

Reply:
Should we make `Open in VS Code` executable even if the directory is not a git one?
The button would be visible only in git repos, but this way a user could assign a shortcut - once we have RichCommands - and use it in other directories as well

Reply:
That's reasonable.